client/lib/state-persistence/worker.js

```diff
@@ -0,0 +1,83 @@
+const getWorker = ( () => {
+	let worker;
+
+	function createWorker() {
+		const blob = new Blob( [ `
+			importScripts( 'https://cdnjs.cloudflare.com/ajax/libs/localforage/1.4.3/localforage.js' );
```

Comment:
might be useful to define this in an actual js file and pull that in using webpack's raw loader. maybe with a `.worker-js` extension or something to avoid the js / babel loader?